bin/s2nc.c

```diff
@@ -411,6 +422,20 @@ int main(int argc, char *const *argv)
             early_data = load_file_to_cstring(optarg);
             GUARD_EXIT_NULL(early_data);
             break;
+        case OPT_PERF_SEND_AMOUNT:
+            errno = 0;
+            perf_send_amount = strtoull(optarg, 0, 10);
+            if (errno == ERANGE) {
+                perf_send_amount = 0;
+            }
```

Comment:
Based on the man page for strtoull, should we be checking that perf_send_amount==ULONG_MAX as well as errno?

Reply:
checking errno should be sufficient, as the man page states that errno is set and the value ULONG_MAX is returned.  however as errno is zeroed before calling, it follows that if errno is ERANGE then the return value is also ULONG_MAX